### PR TITLE
Update running examples in the README

### PR DESCRIPTION
**Pull request by nllong**
#### Any background context you want to provide?
The README file was out of date.

#### What does this PR accomplish?
* Fix the running examples using jm_ipython section.
* Move the setting up of the MBL into the required installation section.

#### How should this be manually tested?
* run through the installation and example simulations.
* Run the python script manually to create the PNG.

#### What are the relevant tickets?
#207

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -22,16 +22,23 @@ The GeoJSON Modelica Translator is in alpha-phase development and the functional
 * (optional/as-needed) Add Python 3 to the environment variables
 * Install `Docker <https://docs.docker.com/get-docker/>`_ for your platform
 * Configure Docker on your local desktop to have at least 4 GB Ram and 2 Cores. This is configured under the Docker Preferences.
-* **Follow first 3 major bullets in Running Simulations section below.**
-* Run :code:`pip install -r requirements.txt`
+* Install the Modelica Buildings Library from GitHub
+    * Clone https://github.com/lbl-srg/modelica-buildings/ into a working directory outside of the GMT directory
+    * Change the directory inside the modelica-buildings repo you just checked out
```
Comment (vtnate) [inline README.rst:27]: Do you mean to instruct someone to move inside the mbl repo before installing git-lfs? This line doesn't make that clear.

**Review by vtnate — APPROVED** [review]

File: README.rst [hunk 2]
```diff
@@ -81,20 +88,19 @@ In some cases, the Level 3 case (translation to Modelica) is a blackbox method (
...
-* Example simulation:
-    * :code:`jm_ipython.sh jmodelica.py spawn_two_building.Loads.B5a6b99ec37f4de7f94020090.building`
-    * :code:`jm_ipython.sh jmodelica.py spawn_two_building/Loads/B5a6b99ec37f4de7f94020090/building.mo`
-* Visualize the results by inspecting the resulting mat file using BuildingsPy.
+* Make sure jm_ipython.sh is in your local path.
+* After running the :code:`py.test` go into the :code:`tests/model_connectors/output` directory
+* Copy jmodelica.py to path :copy:`cp cp ../../../geojson_modelica_translator/modelica/lib/runner/jmodelica.py .`
+* Run examples using either of the the following:
+    * :code:`jm_ipython.sh jmodelica.py spawn_single.Loads.B5a6b99ec37f4de7f94020090.coupling`
+    * :code:`jm_ipython.sh jmodelica.py spawn_single/Loads/B5a6b99ec37f4de7f94020090/coupling.mo`
```
Comment (vtnate) [inline README.rst:101]: There are several warnings at this step, including a notification of deprecated functionality (`cardinality() function-like operator`). Do we care, and if so, is there anything we can do about it?

**Comment by vtnate** [comment]
Example sims run, png is created, looks good.

File: README.rst (hunk 2 above)
Comment (nllong) [reply README.rst:101]: yeah, those warnings are okay. They are Modelica compilation warnings. I'll add a note.

File: README.rst [hunk 3]
```diff
@@ -81,20 +88,20 @@ In some cases, the Level 3 case (translation to Modelica) is a blackbox method (
...
-    * :code:`jm_ipython.sh jmodelica.py spawn_two_building/Loads/B5a6b99ec37f4de7f94020090/building.mo`
-* Visualize the results by inspecting the resulting mat file using BuildingsPy.
+The GeoJSON to Modelica Translator contains a :code:`ModelicaRunner.run_in_docker(...)` method. It is recommended
+to use this method in a python script if needed as it will copy the required files into the correct location. If
+desired, a user can run the simulations manually using JModelica (via Docker). Follow the step below to configure
+the runner to work locally.
+
+* Make sure jm_ipython.sh is in your local path.
+* After running the :code:`py.test` go into the :code:`tests/model_connectors/output` directory
+* Copy jmodelica.py to path :copy:`cp ../../../geojson_modelica_translator/modelica/lib/runner/jmodelica.py .`
```
Comment (vtnate) [inline README.rst:98]: This `:copy:` still doesn't look right on my machine: https://github.com/urbanopt/geojson-modelica-translator/tree/207-docs/readme#running-simulations. Are you sure you didn't mean `:code:`?

**Comment by nllong** [comment]
Crud.. yes, I mean :code:. I saw that double cp and didn’t notice the other issue. Thanks for finding it.

Da: Nathan Moore <notifications@github.com>
Data: giovedì, ottobre29 2020 08:50
A: urbanopt/geojson-modelica-translator <geojson-modelica-translator@noreply.github.com>
Cc: Long, Nicholas <Nicholas.Long@nrel.gov>, Author <author@noreply.github.com>
Oggetto: Re: [urbanopt/geojson-modelica-translator] Update running examples in the README (#208)

@vtnate commented on this pull request.

________________________________

In README.rst<https://gcc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Furbanopt%2Fgeojson-modelica-translator%2Fpull%2F208%23discussion_r514319338&data=04%7C01%7Cnicholas.long%40nrel.gov%7Cd7c1fc38faed476e889308d87c19fddf%7Ca0f29d7e28cd4f5484427885aee7c080%7C0%7C0%7C637395798357744713%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=Uj4%2B0jFes40S3zB0Ns1D1OHLpYJuNDooMvwykLmyUM8%3D&reserved=0>:

> -        * Mac: :code:`brew install git-lfs; git lfs install`

-        * Ubuntu: :code:`sudo apt install git-lfs; git lfs install`

-    * Pull the correct branch with: :code:`git checkout issue1437_district_heating_cooling`

-* Add the Buildings Library path to your MODELICAPATH environment variable (e.g., export MODELICAPATH=${MODELICAPATH}:$HOME/path/to/modelica-buildings).

-* Example simulation:

-    * :code:`jm_ipython.sh jmodelica.py spawn_two_building.Loads.B5a6b99ec37f4de7f94020090.building`

-    * :code:`jm_ipython.sh jmodelica.py spawn_two_building/Loads/B5a6b99ec37f4de7f94020090/building.mo`

-* Visualize the results by inspecting the resulting mat file using BuildingsPy.

+The GeoJSON to Modelica Translator contains a :code:`ModelicaRunner.run_in_docker(...)` method. It is recommended

+to use this method in a python script if needed as it will copy the required files into the correct location. If

+desired, a user can run the simulations manually using JModelica (via Docker). Follow the step below to configure

+the runner to work locally.

+

+* Make sure jm_ipython.sh is in your local path.

+* After running the :code:`py.test` go into the :code:`tests/model_connectors/output` directory

+* Copy jmodelica.py to path :copy:`cp ../../../geojson_modelica_translator/modelica/lib/runner/jmodelica.py .`

This :copy: still doesn't look right on my machine: https://github.com/urbanopt/geojson-modelica-translator/tree/207-docs/readme#running-simulations<https://gcc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Furbanopt%2Fgeojson-modelica-translator%2Ftree%2F207-docs%2Freadme%23running-simulations&data=04%7C01%7Cnicholas.long%40nrel.gov%7Cd7c1fc38faed476e889308d87c19fddf%7Ca0f29d7e28cd4f5484427885aee7c080%7C0%7C0%7C637395798357754669%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=9Sq5iaiUuPMKDIFZ%2B6YX4fLW8BvlriDYDpXF8GTRUY4%3D&reserved=0>. Are you sure you didn't mean :code:?

**Pull request merged**